### DISCUSSION TITLE
Hello and welcome the the Godot Community Games organization!

### DISCUSSION THREAD

**Discussion by GodotCommunityGames** [Godot-Community-Games-Org/.github#1] [Announcements]
Okay I made a discussion section for the org as a whole. Say hi!

**Comment by AnonymousAggregator** [comment]
Hello

**Comment by Welshy92** [comment]
hi!

**Comment by Dragon20C** [comment]
would a new discussion be made for each game, and I think the read me file in the games repo should include a general idea on what the project is going to be about, rpg, fps, etc

**Reply by Welshy92** [reply]
A discussion for separate games would make sense. A discord server with separate channels maybe if everyone uses it.

The README can almost act as a public facing version of a game design document, while including credits, ect...

**Reply by GodotCommunityGames** [reply]
Yes for sure. There will be this overall discussion board, and then each game repo would have it's own discussion board. And yes agreed on the README

**Comment by Emad-Alhaddad** [comment]
##### Hi, I'm excited to participate in this awesome idea. It will be a place for newcomers to see and experiment on different game genres and level of complexity. This will be both fun and educational.

**Reply by GodotCommunityGames** [reply]
Yes exactly. Ideally this will be a community just in general for Godot users of any level to jump in on a game and work on whatever they think they can handle, and be part of a community. A great experience to learn from others and make games together and work in grouips.

**Comment by KaizNike** [comment]
Hello, a good idea I'd think is hosting a poll for what game genre or general game we'd like to emulate.

**Reply by GodotCommunityGames** [reply]
Yep. we can definitely get that going.

**Reply by KaizNike** [reply]
For those linked to this convo, check the Game1 repo for the genre poll, currently roguelikes are in the lead.

**Comment by BentPen** [comment]
Sounds fun let's do it

**Comment by TheBardWhoCodes** [comment]
Hey! Just saw the Reddit post. I've been a software engineer for about 10 years. I've only recently tinkered with game development and Godot but would love to contribute in my free time.

**Reply by GodotCommunityGames** [reply]
Cool, I added you as a member!

**Comment by PickleOnAString** [comment]
heya! been using Godot for a little over 2 years now, excited to see how this develops!

**Comment by Egghuntress** [comment]
Hello all

**Reply by GodotCommunityGames** [reply]
hi! added you as a member.

**Comment by MariuszGaljan** [comment]
Hi everyone, awesome idea with the organization!

I'd love to get involved in exciting game projects.

As for my experience, I'm pretty new to Godot, but I've been working as a Unity developer for about 2 years and since then I switched my career towards web development with ASPNET and stuff. I hope I can contribute somehow :)

**Reply by GodotCommunityGames** [reply]
Awesome. With 2 years of game dev experience I'm sure you'll be able to contribute plenty!

**Comment by gil-berth** [comment]
Awesome project, I'd love to join!

**Reply by GodotCommunityGames** [reply]
Cool, I'll add you as a member tomorrow. Github is limiting how many people I can add in one day. In the meantime join the Discord!

**Reply by GodotCommunityGames** [reply]
added you!
